tests/test_transcription.py

```diff
@@ -64,3 +64,14 @@ def test_transcribe_prerecorded_find_and_replace_list():
     )
     actual_transcript = response["results"]["channels"][0]["alternatives"][0]["transcript"]
     assert actual_transcript == MOCK_TRANSCRIPT.replace("fast", "slow").replace("miss", "snooze")
+
+
+@pytest.mark.asyncio
+async def test_transcribe_prerecorded_file():
+    """
+    Test basic asyncronous pre-recorded transcription.
+    """
+    example_wav_file = os.path.join(CURRENT_DIRECTORY, "the-missile-short.wav")
```

Comment:
Can we avoid committing a 1+ MB audio file to the repo, for the sake of keeping the package lighter-weight for users (even though it's already 170 MB 😬 )? We could submit a URL of a DG-hosted file instead of the audio itself?

Reply:
Would the test then download the file to the user's computer before running the test?

Reply:
We alternatively create a config file that prevents the-missle-short.wav from being included in the release package.

Reply:
I did `AUDIO_URL = "https://static.deepgram.com/examples/Bueller-Life-moves-pretty-fast.wav"` for the other tests. Does it work to keep the pattern of just submitting the JSON to DG without having the audio locally at all?

Reply:
Jason and I discussed separately - this test requires local audio to test the buffer, and we don't have any audio files in the repo yet. Fine as-is.